pom.xml

```diff
@@ -40,6 +40,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.telegram</groupId>
+			<artifactId>telegrambots-spring-boot-starter</artifactId>
```

Comment:
use [teleroute](https://github.com/ArtemGet/teleroute.telegrambots), here is an [example](https://github.com/varya-kot/periods-bot)